applications/GeoMechanicsApplication/tests/cpp_tests/custom_elements/test_interface_element.cpp

```diff
@@ -792,27 +840,53 @@ KRATOS_TEST_CASE_IN_SUITE(TriangleInterfaceElement_LeftHandSideContainsMaterialS
     // Assert
     auto expected_left_hand_side = Matrix{18, 18};
     // clang-format off
+
+    // Since the rotation is about the z-axis (the normal of the triangle) and the two shear
+    // stiffnesses are equal, the left-hand side matrix is equal to the left-hand side of a
+    // non-rotated surface interface element.
+    KRATOS_EXPECT_MATRIX_NEAR(actual_left_hand_side, ExpectedLeftHandSideForTriangleElement(), Defaults::relative_tolerance)
+}
+
+KRATOS_TEST_CASE_IN_SUITE(TriangleInterfaceElement_LeftHandSideContainsMaterialStiffnessContributions_RotatedAlongYAxis,
```

Comment:
Nitpicking, and I'd suggest you to only change it if there are other, more important changes to be made:
```suggestion
KRATOS_TEST_CASE_IN_SUITE(TriangleInterfaceElement_LeftHandSideContainsMaterialStiffnessContributions_RotatedAboutYAxis,
```

Reply:
Done